build.sh

```diff
@@ -222,7 +222,7 @@ case $__HostOS in
         __monoroot=/usr/local
         ;;
     OSX)
-        __monoroot=/Library/Frameworks/Mono.framework/Versions/Current
+        __monoroot=/usr/local
```

Comment:
BTW, I actually didn't intend to commit this change so I'll revert it in a minute, but we might want to provide a mechanism for overriding this path.